src/test/java/core/basesyntax/service/impl/FruitServiceImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.interfaces.FruitService;
+import core.basesyntax.strategy.BalanceOperationStrategy;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.PurchaseOperationStrategy;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitServiceImplTest {
+    private FruitService fruitService;
+    private Storage storage;
+    private Map<String, OperationStrategy> operationStrategies;
+
+    @Before
+    public void setUp() {
+        operationStrategies = new HashMap<>();
+        operationStrategies.put("b", new BalanceOperationStrategy());
+        operationStrategies.put("p", new PurchaseOperationStrategy());
+        fruitService = new FruitServiceImpl(operationStrategies);
+        storage = Storage.getInstance();
+    }
+
+    @Test
+    public void testProcessTransactions_BalanceAndPurchaseOperations() {
```

Comment:
```suggestion
    public void processTransactions_BalanceAndPurchaseOperations_ok() {
```
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/20326858/9d38ef3f-9fde-41c6-9b2d-735ded68f764)